guide/src/main/asciidoc/problems/expiredAccessToken.adoc

```diff
@@ -13,6 +15,7 @@ Authorization: Bearer ds4d2f13sdds2q13qxcgbdf245
 ```
 HTTP/1.1 401 Unauthorized
 Content-Type: application/problem+json
+WWW-Authenticate: Bearer realm="example", error="invalid_token", error_description="The access token is expired."
```

Comment:
In https://github.com/belgif/rest-problem-java/blob/c36e0c9ab0e410b8fd2846d0fb7bfe520a645af0/belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/ExpiredAccessTokenProblem.java#L61 it's error_description="The access token expired". Like the example in the spec. I also did not provide the option to specify the realm. Should I add that?